js-code-guidelines.md

```diff
@@ -10,56 +10,52 @@ These guidelines reflect our shared consensus on protocol and etiquette from wha
 - Tasks (asset pipeline, transpiling, releasing, etc).
 - Dependency management.
 
-Our toolkit for each of these is not set in stone, and we don't plan to halt our constant search for better tools. Get in touch if you've got ideas.
-
-Also, remember:
-
-![Barbossa's warning](assets/CodeIsMoreLikeGuidelines.jpg)
+Our toolkit for each of these is not set in stone, and we don't plan to halt our constant search for better tools. Get in touch if you've got ideas. [These are guidelines and rather then rules](assets/CodeIsMoreLikeGuidelines.jpg).
```

Comment:
wordo: _These are guidelines ~and~ rather then rules_

Reply:
Inception, a wordo inside a wordo! Thanks both! 😀